dustmaps/sfd.py

```diff
@@ -93,43 +93,84 @@ def query(self, coords, order=1):
 
 class SFDQuery(SFDBase):
     """
-    Queries the Schlegel, Finkbeiner & Davis (1998) dust reddening map.
+    Queries maps stored in the same format as Schlegel, Finkbeiner & Davis (1998) dust reddening map, providing results from the Schlegel, Finkbeiner & Davis (1998) dust map by default.
     """
 
     map_name = 'sfd'
     map_name_long = "SFD'98"
 
-    def __init__(self, map_dir=None):
+    def __init__(
+        self, 
+        map_dir=None,
+        whichparent='SFD',
+        which='dust'
```

Comment:
Can we come up with more intuitive names for these keyword arguments? Maybe `map_variant` instead of `whichparent`, and `component` instead of `which`? Additionally, is it possible to infer `whichparent` from `which`? If so, then one option will suffice.

Reply:
Happy to make the variable name change. It is true that which is unique, and specifies whichparent, but I hadn't wanted to complicate things with the logic. The logic is fully specified by the docstring "Should be either ``dust`` (the default), ``i100``, ``i60``, ``mask``, ``temp``, or ``xmap`` with whichparent ``SFD``. Should be ``Beta`` for whichparent ``Synch``. Should be ``Rmap`` for whichparent ``FINK``. Should be ``clean`` for whichparent ``Haslam``" if you wanted to push that change.

Reply:
Some of the logic issues come from a naming convention that Finkbeiner et al changed between the different whichparent files. There is a trailing _4096 for the whichparent SFD files, but none of the others